sonarqube/chart/templates/secret.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+data:
+  admin-password: {{ .Values.global.sonarAdminPasswordB64 }}
+  {{ if ( eq .Values.sonarqube.sonarAuthSaml "true" )}}
+  Saml-Provider-ID: {{ .Values.sonarqube.sonarAuthSamlProviderIdB64 }}
+  Saml-login-url: {{ .Values.sonarqube.sonarAuthSamlLoginUrlB64 }}
+  Saml-cert: {{ .Values.sonarqube.sonarAuthSamlCertficateB64 }}
```

Comment:
Is the casing on purpose?

Reply:
yes, in case you do not want to use SAML and rely on local users you can set the value to false in the confogiration and you will not have any SAML config in your installation

Reply:
Oh I meant upper/lowercase ...

Reply:
Ohh, it is on purpose but not because it is required to be in that way so it can be modified as we need/want.
If this is hard to understand or it is better to set the names diferently we can do it

Reply:
Made the values lowercase